src/ui/sass/modules/_ResultsHeader.scss

```diff
@@ -51,7 +50,7 @@
   &-filterLabel
   {
     display: flex;
-    margin-right: $results-header-spacing * 3 / 8;
+    margin-right: $results-header-spacing / 2;
```

Comment:
This seems like a fairly common quantity in this CSS file, let's make a variable for it

Reply:
I tried creating a $results-header-filters-spacing and then that didn't feel right because it wasn't only for applied filters so then I called it $results-header-horizontal-spacing, but then basically everything except for the padding-top/bottom value was replaced. I feel like this is not particularly more useful than $results-header-spacing but I could be wrong.

Reply:
updated to show what this might look like I'm fine either way